driver/fdo.go

```diff
@@ -34,35 +70,39 @@ func FdoLookupXdgDataDirs() []string {
 	return locationLookup
 }
 
-//FdoLookupApplication looks up an application by name and returns an IconData struct
-func FdoLookupApplication(appName string) *IconData {
-	locationLookup := FdoLookupXdgDataDirs()
+//fdoLookupApplication looks up an application by name and returns an fdoIconData struct
+func fdoLookupApplication(theme string, size int, appName string) *fdoIconData {
+	iconTheme = theme
```

Comment:
Setting globals as a side effect of a "read only" function (such as get, lookup etc) is a bit of a smell.
If this function is called for different themes the results could be undefined... surely these global values are not needed?